spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -18,11 +18,7 @@ CURRENT_DIR="$(pwd)"
 
 MVN="${CURRENT_DIR}/../mvnw"
 
-MVN_VERSION=$($MVN -q \
-    -Dexec.executable=echo \
-    -Dexec.args='${project.version}' \
-    --non-recursive \
-    exec:exec)
```

Comment:
slightly easier way (and more on par with recent maven) to get the version of the project. I also renamed this to `PROJECT_VERSION` as it makes more sense vs `MVN_VERSION`, at least to me.